features/0794-did-rotate/README.md

```diff
@@ -62,13 +66,17 @@ Message Type URI: https://didcomm.org/did-rotate/1.0/ack
 
 This message is still sent to the prior DID to acknowledge the receipt of the rotation. Following messages will be sent to the new DID.
 
-`to_did`: The new DID to be used to identify the **rotating_party**
```

Comment:
For this ACK and Problem Report section it is typical to talk about the two messages being “adopted” by the ACK and Report Problem messages, to include a link and to ensure that the usage is consistent.  I didn’t check on the consistency of use, but I’m guessing it was done :-).

Reply:
added